node-commons/src/assembly/resources/conf/iotdb-common.properties

```diff
@@ -84,6 +84,18 @@
 # Datatype: Double
 # schema_region_per_data_node=1.0
 
+# The policy of extension SchemaRegionGroup for each Database.
+# These policies are currently supported:
+# 1. CUSTOM(Each Database will allocate schema_region_group_per_database RegionGroups as soon as created)
+# 2. AUTO(Each Database will automatically extend SchemaRegionGroups based on the data it has)
+# Datatype: String
+# schema_region_group_extension_policy=AUTO
```

Comment:
Will it be better if we move the parameter `schema_region_group_extension_policy` ahead of the `schema_region_per_data_node`?